linechart.py

```diff
@@ -72,6 +73,132 @@ class GentleValueError(ValueError):
     """
 
 
+class RenderingError(ValueError):
```

Comment:
Nix this exception hierarchy, please. Raise `GentleValueError(i18n.trans(...))` instead.

The error handler should look to `err.args[0]`.

Unit tests should test the I18nMessage.